bin/pocs_shell

```diff
@@ -872,6 +873,11 @@ def parse_options(opts):
         else:
             name, value = opt.split('=')
             name = name.replace('--', '')
+
+            # Make it a number if possible.
+            with suppress(ValueError):
```

Comment:
I can't say I love doing this automatically, i.e. always. I understand that it is simple, but you already have code in do_drift_test that maps one option to an int and the other to a float, and in do_full_drift_test also. How about a flag to allow it to be disabled?

Reply:
Agreed, I'm not too fond of it either which is why I wanted your input. A future developer could be confused as to what is happening. 

I'll look into a flag or some other method, thanks.

Reply:
I've changed this so that anything with an explicit single string remains a string, which I think works better than a flag for not parsing as it allows for some params to be parsed and others not.

Also moved this out of `pocs_shell`. I added some doctests but my local `sphinx` install is giving me segfaults and is sucking too much of my time. :(